pcraster_tools/processing/algorithms/col2map.py

```diff
@@ -69,8 +70,8 @@ def initAlgorithm(self, config=None):  # pylint: disable=missing-function-docstr
             )
         )
 
-        datatypes = [self.tr('Boolean'), self.tr('Nominal'), self.tr('Ordinal'), self.tr('Scalar'),
-                     self.tr('Directional'), self.tr('LDD')]
+        datatypes = [self.tr('Boolean'), self.tr('Nominal (small)'), self.tr('Nominal (large)'), self.tr('Ordinal (small)'), 
```

Comment:
Changing the index of existing options has the potential to break existing models or scripts -- are you ok with that?

Reply:
Good point. Can this maybe be solved by adding a checkbox for "large" and "radians" instead?
Default not checked, so it uses small integers or degrees. Checked for large integers (nominal/ordinal) or radians (directional).

Reply:
What about rearranging it so that the two "large" entries are added at the end? Then existing scripts using nominal/ordinal would still map across to nominal/ordinal types (or vice versa, and add the "small" types to the end)

Reply:
Sounds okay to add the "large" entries for nominal/ordinal to the end, followed by directional in radians.

Reply:
@nyalldawson Please check the updated script.